docs/configuration/registration/objects.rst

```diff
@@ -0,0 +1,52 @@
+.. _configuration_registration_objects:
+
+============
+Objecten API
+============
+
+The Objecten API is a suite of REST based object registration services. Open Forms can be
+configured to access these API's to register form submissions.
+
```

Comment:
@stevenbal could you add an example ProductAanvraag Objecttype here?

Reply:
@joeribekker do you mean an example of a ProductAanvraag object, or the entire objecttype? This is what a ProductAanvraag could look like (`data` contains the actual form data the user entered)

```json
{
    "data": {
         "naam": "Jan Jansen",
         "omschrijving": "Ik heb een vraag over mijn paspoort",
         "telefoonnummer": "0612345678"
    },
    "type": "terugbelnotitie",
    "bsn": "111222333",
    "pdf_url": "https://openzaak.nl/documenten/api/v1/enkelvoudiginformatieobjecten/230bab4a-4b51-40c6-91b2-f2022008a7f8", 
    "attachments": [],
    "submission_id": "a43e84ac-e08b-4d5f-8d5c-5874c6dddf56"
}
```

Reply:
Thanks, I copied the whole structure and noted the top-level is meta-data of the submission and the data element has the form values.